src/main/java/io/strimzi/test/container/StrimziKafkaContainer.java

```diff
@@ -32,120 +34,61 @@
  */
 public class StrimziKafkaContainer extends GenericContainer<StrimziKafkaContainer> {
 
+    // class attributes
     private static final Logger LOGGER = LogManager.getLogger(StrimziKafkaContainer.class);
-
-    /**
-     * Default Kafka port
-     */
-    public static final int KAFKA_PORT = 9092;
-    /**
-     * Default ZooKeeper port
-     */
-    public static final int ZOOKEEPER_PORT = 2181;
-
     private static final String STARTER_SCRIPT = "/testcontainers_start.sh";
+    private static final KafkaVersionService LOGICAL_KAFKA_VERSION_ENTITY;
+    private static int kafkaDynamicKafkaPort;
```

Comment:
No need for two Kafkas surely, `dynamicKafkaPort` or `kafkaDynamicPort`?

Reply:
Och, we need it :)) moving it to instance variable... )